core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1503,8 +1503,13 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
    */
   @DeveloperApi
   def getRDDStorageInfo: Array[RDDInfo] = {
+    getRDDStorageInfo( _ => true)
+  }
+
+  @DeveloperApi
+  def getRDDStorageInfo(filter: RDD[_] => Boolean): Array[RDDInfo] = {
```

Comment:
I would just make this `private[spark]`. Let's try to limit the number of things we expose.